.github/workflows/rust.yml

```diff
@@ -40,6 +40,38 @@ jobs:
       - run: cargo test --all-targets
       - run: cargo run --example stress-test
 
+  test-only:
```

Comment:
instead of blowing up CI with even more checks, would y'all be open to moving the const and the test to a `ffi_utils.rs` mod or something that we keep platform agonstic?

Reply:
Yes, that would make sense.

Reply:
done